src/test/arith_uint256_tests.cpp

```diff
@@ -564,4 +564,12 @@ BOOST_AUTO_TEST_CASE( getmaxcoverage ) // some more tests just to get 100% cover
     CHECKBITWISEOPERATOR(R1,~R2,&)
 }
 
+BOOST_AUTO_TEST_CASE( blob88_increment )
+{
+    blob88 bzero(0);
```

Comment:
This name shadows a [function in glibc](https://man7.org/linux/man-pages/man3/bzero.3.html), which I guess is fine but I would rename it to `b_zero` (and `b_one`).